### PR TITLE
Misc minor UI cleanup

### PR DESCRIPTION
**Pull request by tomitm**
@mitchhentges pointed me towards this project, so I had a look around!

Noticed a few things that could use fixing:

- At first, wasn't completely sure what the random :no_entry_sign: in the bottom right did. Hovered over... nothing. So, I added a tooltip! The rest of the footer didn't have tooltips either, and there were a bunch of toggles. Best not to assume everyone realises they're buttons. So, tooltip all the things!
image

- While I was in the footer, I realised it could take advantage of React 0.14's [stateless components](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components). I was going to change `Throttle` as well, since it seems like it should be stateless as well, but that wasn't immediately working. I'll try to poke at it later, since I believe all the state should be higher up.

- While inspecting a request, the toolbar that contains close has an invisible overlay that prevents you from clicking or scrolling on elements underneath. That's fixed as well.

- Speaking of the close button, I noticed it had some alignment issues (#85) and fixed that as well. Also gave it a bit of `margin` from the sides so it didn't look awkward.
image

- In the list of requests, you can't really tell which one you're inspecting. So when you're looking at a bunch of similar or even identical requests, you're sorta out of luck. No longer an issue, as the active request is now **bold**.
image

Hopefully this isn't too many fixes for a single PR, if so, I can split it up.

### CODE REVIEW

File: src/component/footer/requestCount.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const {object, func} = React.PropTypes;
+
+export const RequestCount = ({requestData, clearRequests}) => {
+  const {filteredCount, totalCount} = requestData;
```
Comment (mitchhentges) [inline src/component/footer/requestCount.js]: Ooh, ES6 string-template thingers, nice

File: src/component/footer/requestCount.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  return <div className="request-count">
+    {`Requests: ${filteredCount} / ${totalCount}`}
```
Comment (mitchhentges) [inline src/component/footer/requestCount.js]: Ooh, ES6 string-template thingers, nice

**Comment by mitchhentges** [comment]
:+1:, but a bigger PR, so I'll set @nerdbeere merge

File: src/component/footer/requestCount.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (mitchhentges) [inline src/component/footer/requestCount.js]: Could you name this file in `hyphen-case` rather than `camelCase`?

Comment (nerdbeere) [inline src/component/footer/requestCount.js]: +1

File: src/component/footer.js [hunk 4]
```diff
@@ -1,43 +1,25 @@
 import React from 'react';
-import Throttle from './footer-throttle';
+import {CacheButton} from './footer/cache';
+import Throttle from './footer/throttle';
+import {RequestCount} from './footer/requestCount';
 
 const {func, object} = React.PropTypes;
 
-export default class Footer extends React.Component {
+export const Footer = (props) => {
```
Comment (nerdbeere) [inline src/component/footer.js]: Doesn't it make sense to still export it as default here?

File: src/index.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import hoxy from 'hoxy';
 import TitleBar from './component/titlebar';
-import Footer from './component/footer';
+import {Footer} from './component/footer';
```
Comment (nerdbeere) [inline src/index.js]: If you export footer as default you wouldn't have to use destructuring here

**Comment by nerdbeere** [comment]
Awesome, thanks for your contribution!

File: src/component/footer.js (hunk 4 above)
Comment (mitchhentges) [inline src/component/footer.js]: The syntax `export default const ...` isn't valid, according to https://esdiscuss.org/topic/why-is-export-default-var-a-1-invalid-syntax.

Instead, you'd need to:
```
const Footer = (props) => {
...
}

export default Footer

Comment (nerdbeere) [inline src/component/footer.js]: Which would make more sense to me :)

Comment (mitchhentges) [inline src/component/footer.js]: Ah, just making sure that you got that :+1:

File: src/component/footer/requestCount.js (hunk 3 above)
Comment (tomitm) [inline src/component/footer/requestCount.js]: I should also point out the subtle `/footer/` folder addition - is that okay?

File: src/component/footer.js (hunk 4 above)
Comment (tomitm) [inline src/component/footer.js]: Yeah, I ran into the invalid part Mitch pointed out, but didn't realise there was a workaround... weird that `export default class` is valid yet `export default const` is not. Special case for class, I guess? ¯\\\_(ツ)\_/¯

Fixed, but now I want to go find out why that's the case...

**Comment by mitchhentges** [comment]
Looks real :+1:. Give master a merge, though, so we can verify that the build won't fail.

**Pull request merged**